protocol/contracts/v2/strategies/AbstractStrategy.sol

```diff
@@ -211,4 +211,22 @@ abstract contract AbstractStrategy is ITempleStrategy, TempleElevatedAccess {
         _setTokenAllowance(token, spender, amount);
         emit TokenAllowanceSet(address(token), spender, amount);
     }
+
+    /**
+     * @notice A hook which is called by the Treasury Reserves Vault when the debt ceiling
+     * for this strategy is updated
+     * @dev by default it's a no-op unless the strategy implements `_debtCeilingUpdated()`
+     */
+    function debtCeilingUpdated(IERC20 token, uint256 newDebtCeiling) external override {
```

Comment:
nit: function arguments are not used here or in the child contracts

Reply:
Correct it's to future proof for future strategies which might need it